packages/airnode-node/src/handlers/sign-oev-data.ts

```diff
@@ -46,33 +22,14 @@ export async function signOevData(
 
     return { ...beacon, templateId };
   });
-  const beaconIds = beaconsWithTemplateId.map((beacon) => deriveBeaconId(beacon.airnodeAddress, beacon.templateId));
-  // We are computing both update value and data feed ID in Airnode otherwise it would be possible to spoof the signature.
-  const dataFeedId = beaconIds.length === 1 ? beaconIds[0] : deriveBeaconSetId(beaconIds);
-  const timestamp = calculateUpdateTimestamp(validUpdateTimestamps);
-  const updateValue = calculateMedian(validUpdateValues);
-  const encodedUpdateValue = ethers.utils.defaultAbiCoder.encode(['int256'], [updateValue]);
-  const oevUpdateHash = ethers.utils.solidityKeccak256(
-    ['uint256', 'address', 'address', 'bytes32', 'bytes32', 'uint256', 'bytes', 'address', 'uint256'],
-    [
-      chainId,
-      dapiServerAddress,
-      oevProxyAddress,
-      dataFeedId,
-      updateId,
-      timestamp,
-      encodedUpdateValue,
-      bidderAddress,
-      bidAmount,
-    ]
-  );
-
   const beaconsToSign = beaconsWithTemplateId.filter((beacon) => beacon.airnodeAddress === airnodeAddress);
   const goSignatures = await go(() =>
     Promise.all(
       beaconsToSign.map((beacon) =>
         airnodeWallet.signMessage(
-          ethers.utils.solidityKeccak256(['bytes32', 'bytes32'], [oevUpdateHash, beacon.templateId])
+          ethers.utils.arrayify(
```

Comment:
This was a bug before. We need to be careful about this as it was quite tricky to debug.

Reply:
So sorry about this :see_no_evil: